src/WebEid.Security/Validator/CertValidators/SubjectCertificateTrustedValidator.cs

```diff
@@ -39,14 +40,18 @@ public SubjectCertificateTrustedValidator(ICollection<X509Certificate2> trustedC
         }
 
         /// <summary>
-        /// Validates that the user certificate from the authentication token is signed by a trusted certificate authority.
+        /// Checks that the user certificate from the authentication token is valid and signed by
+        /// a trusted certificate authority.Also checks the validity of the user certificate's
```

Comment:
Minor missing space:
```suggestion
        /// a trusted certificate authority. Also checks the validity of the user certificate's
```

Reply:
Fixed with latest commit